api/about.json

```diff
@@ -0,0 +1,9 @@
+{
+ "0.01": [
```

Comment:
Apart from @CuriousLearner's comments, make sure to fix the version. We'll have a version check.